pandas/tseries/resample.py

```diff
@@ -1189,13 +1190,18 @@ def _get_time_delta_bins(self, ax):
             raise TypeError('axis must be a TimedeltaIndex, but got '
                             'an instance of %r' % type(ax).__name__)
 
+        if len(ax) > 0 and all(ax._isnan):
```

Comment:
so this is *all* data is nan. Hmm, I would just give a better error message then.